src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -250,12 +255,37 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
       const rewardsAmountsCryptoBaseUnit: [string] = [
         stakedAmountCryptoBaseUnitIncludeRewards.minus(stakedAmountCryptoBaseUnit).toFixed(),
       ]
+      let maturity: number | undefined = undefined
+
+      if (stakingOpportunityId === thorchainAssetId && accountPosition.last_add_height) {
+      }
+
+      const maybeMaturity = await (async () => {
+        if (stakingOpportunityId !== thorchainAssetId) return {}
+        // note undefined over missing property here, i.e this is a runepool position and explicitly undefined
+        if (!accountPosition.last_add_height) return
+
+        const blockParams = new URLSearchParams({
+          height: accountPosition.last_add_height?.toString(),
+        })
+
+        const { data: blockDetails } = await axios.get<ThorchainBlock>(
+          `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/block?${blockParams}`,
+        )
+
+        maturity =
```

Comment:
```suggestion
        const maturity =
```

Reply:
Fixed in [60e41ac](https://github.com/shapeshift/web/pull/7472/commits/60e41aceb721592767aac449c8b1671168120425)